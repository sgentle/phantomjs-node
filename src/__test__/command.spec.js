import Command from '../command';

describe('Command', () => {
    it('id to be randomly generated', () => {
        expect(new Command('test').id).toEqual('85a86bea0cff860c343e5c9b83381e2e');
    });

    it('.target to be set correctly', () => {
        expect(new Command('abc').target).toEqual('abc');
    });

    it('JSON.stringify(command) to be valid json', () => {
        expect(JSON.stringify(new Command('target', 'name'))).toMatchSnapshot();
    });
});
