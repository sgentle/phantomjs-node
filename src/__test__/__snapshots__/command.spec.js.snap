exports[`Command JSON.stringify(command) to be valid json 1`] = `"{\"target\":\"target\",\"name\":\"name\",\"params\":[],\"deferred\":null,\"id\":\"6c7826bdd79d36539e6dc9c358b222bb\"}"`;
